vector/src/main/scala/geotrellis/vector/reproject/package.scala

```diff
@@ -3,6 +3,8 @@ package geotrellis.vector
 import geotrellis.proj4._
 
 package object reproject {
+  type DI = DummyImplicit
```

Comment:
Looks like this was needed when trying an overload but not with renames of apply?